src/server/main.py

```diff
@@ -23,25 +25,35 @@
 @app.get("/")
 async def model(request: Request) -> Response:
     """serve the main model visualization"""
-    return templates.TemplateResponse("model.html", {"request": request})
+    return templates.TemplateResponse(
+        "model.html", {"request": request, "STATIC_URL": STATIC_URL}
```

Comment:
What about having a function:

```
def get_context(request: Request, **kwargs):
  return {"request": request, "STATIC_URL": STATIC_URL, **kwargs}

...
return templates.TemplateResponse("model.html", get_context(request, message=message, ...))
```

so it doesn't violate Don't Repeat Yourself principle?

Reply:
Yep, great idea.